pandasai/helpers/optional.py

```diff
@@ -10,10 +10,10 @@
 import warnings
 from typing import TYPE_CHECKING, List
 
+import pandas as pd
```

Comment:
To ensure compatibility with modin, pandas **must** be imported from `pandasai.pandas`
```suggestion
import pandasai.pandas as pd
```